README.md

```diff
@@ -467,18 +471,6 @@ NNCF is also available via [conda](https://anaconda.org/conda-forge/nncf):
 conda install -c conda-forge nncf
 ```
 
-### System requirements
-
-- Ubuntu\* 18.04 or later (64-bit)
-- Python\* 3.9 or later
-- Supported frameworks:
-  - PyTorch\* >=2.4, <2.6
-  - TensorFlow\* >=2.8.4, <=2.15.1
-  - ONNX\* ==1.17.0
-  - OpenVINO\* >=2022.3.0
-
-This repository is tested on Python* 3.10.14, PyTorch* 2.5.0 (NVidia CUDA\* Toolkit 12.4) and TensorFlow* 2.12.1 (NVidia CUDA\* Toolkit 11.8).
```

Comment:
Why do you want to remove it? We've added it specifically and I thought it was useful as it shows at least versions we're sure about.

Reply:
We are always testing on default environment for PT and TF, no any reason to duplicate this information in nncf readme.
Actual version of backends collects in instalation.md 
It's duplication information from instalation.md.

Reply:
it's incorresct statement that we are always testing on default environment for PT and TF. I would suggest to have place where the CUDA version will be defined explicitly.

Reply:
I'm ok to add it to instalation.md and refer it from here then

Reply:
Updated, please check